gsoc_final_post/inigo_tellaetxe_gsoc_2024_final_report.rst

```diff
@@ -0,0 +1,226 @@
+
+    * After training for 64 epochs to see the progress and tendency of the model, we observed that the best reconstruction loss was very high, whereas the adversarial loss components were relatively low in comparison. I suspected that something was off with the generated samples, so I went on to check the training data reconstruction. To no surprise, the model generated the exact same sample every time, suggesting that the model had fallen into mode collapse, a typical problem with adversarial training.
+
+Objectives in Progress
+----------------------
+
+* Solving the mode collapse issue in the AAE model using two strategies:
+
+  * Implementing a more complex discriminator architecture, with more layers and units, to increase the capacity of the model to distinguish between real and generated samples.
+  * Implementing a `Wasserstein loss <http://arxiv.org/abs/1701.07875>`_ function, which is known to be more stable than the original GAN loss function, and to prevent mode collapse.
+
+* Validating the AAE with the FiberCup data.
+* Implementing and integrating the attribute-based regularization component inside the working AAE architecture.
+
+Future Work
+-----------
+
+I plan to continue with this project until I arrive to a satisfactory solution to generate synthetic human tractograms with tuneable age and neurodegeneration status properties. This project has not only been a great learning experience, but it has also been a great challenge and a great addition to my PhD research, so I will for sure continue until I succeed.
+
+To complete the work, I will need to:
+
+* Complete the objectives in progress.
+* Train and validate the models with human tractograms.
+* Integrate the models into DIPY.
+
+Pull Requests
+-------------
+Because the work is still in progress, no integration of the final version was done yet. However, a draft Pull Request was created to show the progress of the work in the main DIPY repository. For now, it contains the first validated AE, together with tests for the model class. `LINK TO THE PR <https://github.com/dipy/dipy/pull/3328>`_
+
+Timeline
+--------
+
++------------+-------------------------------------------------------------+------------------------------------------------------------------------------------------------------------------------------+
```

Comment:
You could implement list-table directive, will give u less trouble. Have a look here 

https://github.com/dipy/dipy.org/pull/66/files#diff-b5c486bccd768b5473dd16ece90b3a9026925ed7ac5a82faac5c2e1123394ad6R422

Reply:
Thank you so much for this! Again, I suck at formatting and the table gave me a trillion headaches, so I appreaciate it